ibm/mas_devops/roles/cp4d_upgrade/defaults/main.yml

```diff
@@ -0,0 +1,44 @@
+---
+
+mas_instance_id: "{{ lookup('env', 'MAS_INSTANCE_ID') }}"
+mas_namespace: "mas-{{ mas_instance_id }}-core"
+mas_upgrade_supported_channels: ['maint86xdev', '8.6.x']
```

Comment:
remove dev channel?

Reply:
yeah, i think at this point we can remove it safely.